warehouse/query-core/src/main/java/datawave/query/transformer/GroupingTransform.java

```diff
@@ -437,18 +437,19 @@ private void getListKeyCounts(Entry<Key,Document> entry) {
         int longest = this.longestValueList(fieldToFieldWithContextMap);
         for (int i = 0; i < longest; i++) {
             Collection<GroupingTypeAttribute<?>> fieldCollection = new HashSet<>();
-            String currentGroupingContext = "";
```

Comment:
I'd pull this `currentGroupingContext` var into a method since you're replicating what is done in [line 330](https://github.com/NationalSecurityAgency/datawave/blob/8ba03b751d64f3545ed89fe520d074a2bc2ef836/warehouse/query-core/src/main/java/datawave/query/transformer/GroupingTransform.java#L330)

Reply:
Done in the latest commit.